beep-beep/jobs.md

```diff
@@ -0,0 +1,1136 @@
+
+### Beanstalk Driver
+
+Beanstalk is a simple and fast general purpose work queue. To install Beanstalk,
+you can use the [local queue server](https://github.com/beanstalkd/beanstalkd)
+or run the server inside [AWS Elastic](https://aws.amazon.com/elasticbeanstalk/). 
+You can choose any option that is convenient for you.
+
+Setting up the server is similar to setting up AMQP and requires specifying the
+connection in the `"beanstalk"` section of your RoadRunner configuration file.
+
+```yaml
+beanstalk:
+  addr: tcp://127.0.0.1:11300
+```
+
+After setting up the connection, you can start using it. Let's take a look at 
+the complete config with all the options for this driver:
+
+```yaml
+beanstalk:
+  # Optional section.
+  # Default: tcp://127.0.0.1:11300
+  addr: tcp://127.0.0.1:11300
+
+  # Optional section.
+  # Default: 30s
+  timeout: 10s
+
+jobs:
+  pipelines:
+    # User defined name of the queue.
+    example:
+      # Required section.
+      # Should be "beanstalk" for the Beanstalk driver.
+      driver: beanstalk
+      
+      # Optional section.
+      # Default: 10
+      priority: 10
+
+      # Optional section.
+      # Default: 1
+      tube_priority: 1
+      
+      # Optional section.
+      # Default: default
+      tube: default
+
+      # Optional section.
+      # Default: 5s
+      reserve_timeout: 5s
+```
+
+These are all settings that are available to you for configuring this type of 
+driver. Let's take a look at what they are responsible for:
+- `priority` - Similar to the same option in other drivers. This is queue 
+  default priority for for each task pushed into this queue if the priority 
+  value for these tasks was not explicitly set.
+
+- `tube_priority` - The value for specifying the priority within Beanstalk is
+  the internal priority of the server. The value should not exceed `int32` size.
+
+- `tube` - The name of the inner "tube" specific to the Beanstalk driver.
+
+### SQS Driver
+
+[Amazon SQS (Simple Queue Service)](https://aws.amazon.com/sqs/) is an
+alternative queue server also developed by Amazon and is also part of the AWS
+service infrastructure.
+
+However, unlike Beanstalk, it is located entirely on Amazon servers, so the
+connection configuration will differ from the one we are already used to for
+AMQP and Beanstalk. In order to get acquainted with how to create a SQS server, 
+just use the [ready-made documentation](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-configuring.html).
+
+Please also note that although installation as a server using AWS services is
+permissible, however, it is possible to install this server locally by
+downloading the corresponding executable file from Amazon servers and running 
+it using Java (you will also need the installed Java Runtime Environment):
+
+```sh
+wget https://s3-eu-west-1.amazonaws.com/softwaremill-public/elasticmq-server-1.2.0.jar
+
+// Where "custom.conf" is your own config file
```

Comment:
I guess that will be better to point the user to the existing `Dockerfile` with all configuration needed. Or, provide both, local and dockered setup with the sample of the `custom.conf` : https://github.com/spiral/roadrunner/blob/master/tests/env/custom.conf

Reply:
I don’t think it makes sense to write down the complete installation instructions. This example is only needed to show that the SQS is available for local installation too, in addition to being used in AWS.

So yeah, I think a `Dockerfile` link will suffice.

However, I am confused by the fact that it is located in the `tests` directory, which may confuse users and they may think that this use case is only possible for development.

Reply:
P.S. `yaml` is not a valid Dockerfile extension

Reply:
kk. I corrected this description by simply providing a link to the repository with the project.